### DISCUSSION TITLE
Dream2nix is awesome!

### DISCUSSION THREAD

**Discussion by yajo** [nix-community/dream2nix#604] [General]
Hello folks!

After trying many ways to make my project work, dream2nix (v1) is definitely the most awesome way to do it.

I just wanted to say thank you. This really makes nix very much more practical. I hope soon more 2nix tools migrate to d2n format and this just keeps growing up.

Thanks!

**Comment by leonardp** [comment]
I second that!
After fiddling around with pnpm2nix node2nix buildNpmPackage etc. ... dream2nix feels like magic!
I am super hyped about v1 after watching the presentation on youtube.

Thanks!
